tox.ini

```diff
@@ -65,4 +65,12 @@ deps=
 commands=
     python -m build
     twine check dist/*.tar.gz dist/*.whl
-    check-python-versions --only pyproject.toml,.github/workflows/test.yml
\ No newline at end of file
+    check-python-versions --only pyproject.toml,.github/workflows/test.yml
+
+[testenv:docs]
+description = sphinx docs
+basepython = python3.11
+deps =
+    -r docs/requirements.txt
+commands =
+    sphinx-build -M html docs docs/_build -j auto --keep-going {posargs:--fail-on-warning --fresh-env -n}
```

Comment:
The `--fail-on-warning` flag makes this PR fail to build for me locally because there is no `docs/_static` directory. You could create that directory with an empty file named `.gitkeep` to resolve that.